drakrun/drakrun/drakpdb.py

```diff
@@ -324,7 +324,7 @@ def fetch_pdb(pdbname, guidage, destdir='.'):
 
             with tqdm(total=total_size, unit='iB', unit_scale=True) as pbar:
                 with open(dest, "wb") as f:
-                    for chunk in res.iter_content(chunk_size=1024 * 8):
+                    for chunk in res.iter_content(chunk_size=1024 * 1024 * 8):
```

Comment:
This change will help as the download speed will be capped at 8Mbps instead of 8Kbps now

Reply:
I don't think this caps at "per second", rather "at a time". As we have bigger buffers => less loop iterations => less system calls, so it has a chance to work faster.

Reply:
Logical flaw on my side it seems, thanks for the correction 😅